app/Main.hs

```diff
@@ -145,3 +145,17 @@ main = do
     Just HapRollback -> rollback hapConfiguration
 
     Nothing -> hapHelpAction Nothing
+
+splitInColon :: String -> (String, String)
```

Comment:
Thanks @CristhianMotoche for making this change. The only thing I'm not crazy about is the conditional check to see if the host is `null`. What do you think about giving this the following return type, against which we could pattern match?

```haskell
splitInColon :: String -> (String, Maybe String)
```

At least that's what initially came to mind for me. @jpvillaisaza do you have any other ideas?